lib/tasks/lint.rake

```diff
@@ -1,3 +1,5 @@
+task default: :lint
```

Comment:
How will this work with the configuration in the Rakefile?

https://github.com/alphagov/support/blob/master/Rakefile#L9

Reply:
These will still run, I had to do it this way here https://github.com/alphagov/finder-frontend/pull/814. I think because it didn't find the lint task when I added it to the Rakefile.